pkgs/by-name/ta/tangara-cli/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+
+  gdk-pixbuf,
+  glib,
+  graphene,
+  gtk4,
+  libadwaita,
+  pango,
+  pkg-config,
+  udev,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tangara-cli";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "haileys";
+    repo = "tangara-companion";
+    tag = "v${version}";
+    hash = "sha256-pTE+xlXWIOOt1oiKosnbXTCLYoAqP3CfXA283a//Ds0=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-C7Q3Oo/aBBH6pW1zSFQ2nD07+wu8uXfRSwNif2pVlW0=";
+
+  nativeBuildInputs = [
+    glib
+    pkg-config
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    glib
+    graphene
+    gtk4
+    libadwaita
+    pango
+    udev
+  ];
+
+  buildPhase = "cargo --locked build -r -p tangara-cli";
+  installPhase = ''
+    mkdir -p $out/bin;
+    cp -r ./target/release/tangara $out/bin/;
+  '';
```

Comment:
If you use `buildAndTestSubdir` you should be able to drop the extra dependencies - from a quick test I think you only need `pkg-config` and `udev`.

```
buildAndTestSubdir = "crates/tangara-cli";
```

Reply:
Thanks for the suggestion, I had no idea about that option! I concur with your assessment, it looks like it builds and runs with only `pkg-config` and `udev` so I've updated the PR accordingly.